lib/dma.h

```diff
@@ -83,6 +83,9 @@ struct vfu_ctx;
 
 typedef struct {
     dma_addr_t dma_addr;        // DMA address of this region
+    uint32_t prot;              // memory protection of the mapping
```

Comment:
It's `int` in `mmap(2)`.

Reply:
struct vfio_user_dma_region.prot is uint32_t